src/Native/build.proj

```diff
@@ -96,12 +96,16 @@
                                      AND '%(NativePackageAsset.Identity)' != '$(PlaceholderFile)'"
                           Include="@(NativePackageAsset->'%(RootDir)%(Directory)%(Filename)$(NativeLibSymbolExtension)')" />
     </ItemGroup>
-
+	
     <ItemGroup>
-      <NativePackageAsset Include="$(NativeAssetsBuiltPath)\$(NativeLibPrefix)MklImports$(NativeLibExtension)"
-                          RelativePath="Microsoft.ML.HalLearners\runtimes\$(PackageRid)\native" /> 
+      <NativePackageAsset Include="$(NativeAssetsBuiltPath)\$(NativeLibPrefix)MklProxyNative$(NativeLibExtension)"
```

Comment:
This line needs to go above the previous ItemGroup.  The reason this ItemGroup exists down here is because the one above adds the symbols for the assemblies we produce in this repo.  Since `MklImports` is not produced by this repo, we have no symbols for it.  But `MklProxyNative` IS produced by this repo, so we need to package the `$(NativeLibSymbolExtension)` files as well.